microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -412,3 +412,12 @@ Issued: 2024-01-17
 {product-title} release 4.14.9, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:0206[RHBA-2024:0206] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:0204[RHSA-2024:0204] advisory.
 
 For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
+
+[id="microshift-4-14-10-dp"]
+=== RHSA-2024:0292 - {microshift-short} 4.14.10 bug fix update and security advisory
```

Comment:
The title here says "bug fix update and security advisory" while previous titles just say "bug fix update advisory" though the content looks very similar  (the previous ones also have "includes security updates"). Maybe consider making them consistent one way or the other? Not a deal-breaker, so your call.

Reply:
@mramendi They are different advisories. This is an RH_S_A as opposed to an RH_B_A. The "BA" advisories don't have security updates, they are intended for bug fixes (and sometimes, "enhancements").